conans/test/functional/toolchains/cmake/test_ninja.py

```diff
@@ -79,7 +79,7 @@ def test_locally_build_linux(build_type, shared, client):
     assert "main: {}!".format(build_type) in client.out
     client.run("install hello/1.0@ -g=deploy -if=mydeploy {}".format(settings))
     ldpath = os.path.join(client.current_folder, "mydeploy", "hello", "lib")
-    client.run_command("LD_LIBRARY_PATH='{}' ./mydeploy/hello/myapp".format(ldpath))
+    client.run_command("LD_LIBRARY_PATH='{}' ./mydeploy/hello/bin/myapp".format(ldpath))
```

Comment:
This is necessary because of the changes at [conans/test/assets/cmake.py](https://github.com/conan-io/conan/pull/10706/files#diff-7f37d28a6c996b75761c966c3f13f86d16fe007bd5c89e8bca57679202ac61e0)
mostly:
```
{% if appsources %}
        install(TARGETS {{appname}} DESTINATION ".")
```